nav2_behavior_tree/plugins/condition/globally_updated_goal_condition.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2021 Joshua Wallace
```

Comment:
You added a file ` tools/navigate_w_replanning_only_if_path_becomes_invalid.png.png `, that should be removed, but that would be great to add to the Nav2 docs PR. You could add a new page to https://navigation.ros.org/behavior_trees/index.html with the walk through (albeit simple)

Reply:
This image should still be removed from the branch

Reply:
Add as a comment in this PR but remove from source code

Reply:
Removed

Reply:
Still there...

Reply:
Or you didn't push updates?

Reply:
Sorry not yet! 

Just working through the comments now.

Reply:
No worries :laughing: I was just too quick. I'll check back tomorrow afternoon